examples/upbound-aws-provider/composite-resources/s3-irsa/helm-chart/templates/claim.yaml

```diff
@@ -11,5 +11,5 @@ metadata:
 spec:
   resourceConfig:
     providerConfigName: aws-provider-config
-    region: us-east-1
+    region: us-west-2
```

Comment:
Let's leave these for now. Out of scope for this PR.

Reply:
Will revert change. Included in error.

Reply:
change made